LeetcodeProblems/Algorithms/Time_Needed_Rearrange_Binary_String.js

```diff
@@ -36,12 +36,14 @@ s[i] is either '0' or '1'.
  * @param {string} s
  * @return {number}
  */
- var secondsToRemoveOccurrences = function(s) {
+var secondsToRemoveOccurrences = function(s) {
   let result = 0;
-  while (true) {
-    const replaced = s.replaceAll('01', '10');
+  for(;;) {
```

Comment:
🤔 is this new? 🤔

Reply:
Yeah so ESLint triggers lint errors on `while(true)` because of the constant condition.
`for(;;)` is allowed though and is seen as a workaround to this (silly) restriction.
Refer [this](https://stackoverflow.com/questions/37466508/why-does-eslint-trigger-lint-errors-on-whiletrue-using-fibers).
We would probably need to add an exception to disable that in the workflow if we would like to disable the check for this. I'm not very sure of that right now but we can have a look at this later.

Reply:
oh this is super confusing 😢 but ok for now.